backend/metadata/databases/default/tables/tables.yaml

```diff
@@ -11,18 +11,18 @@
 - "!include public_EnrollmentStatus.yaml"
 - "!include public_Event.yaml"
 - "!include public_Instructor.yaml"
+- "!include public_Location.yaml"
 - "!include public_Mail.yaml"
-- "!include public_MailLog.yaml"
```

Comment:
Why does this file still exists? Can we delete it?

Reply:
Not sure why hasura itself doesnt delete these when the tables are removed. deleted them manually now. hope it doesnt cause problerms